LoggingKata/Program.cs

```diff
@@ -16,14 +16,43 @@ static void Main(string[] args)
 
             var lines = File.ReadAllLines(csvPath);
 
-            logger.LogInfo($"Lines: {lines[0]}");
-
             var parser = new TacoParser();
 
             var locations = lines.Select(parser.Parse).ToArray();
 
-            // TODO:  Find the two Taco Bells that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            ITrackable locA = null;
+            ITrackable locB = null;
+            ITrackable locX = null;
+            ITrackable locY = null;
+
+            double distance = 0;
+
+            for (int i = 0; i < locations.Length; i++)
+            {
+                locA = locations[i];
+                GeoCoordinate CorA = new GeoCoordinate(locA.Location.Latitude, locA.Location.Longitude);
+
+                for (int x = i + 1; x < locations.Length; x++)
+                {
+                    locB = locations[x];
+                    GeoCoordinate CorB = new GeoCoordinate(locB.Location.Latitude, locB.Location.Longitude);
+
+                    double currentDistance = CorA.GetDistanceTo(CorB);
+
+                    if(currentDistance > distance)
+                    {
+                        distance = currentDistance;
+                        locX = locA;
+                        locY = locB;
+                    }
+                }
+            }
+
+            Console.WriteLine(locX.Name);
+            Console.WriteLine(locY.Name);
+            Console.WriteLine(distance);
```

Comment:
Consider adding units to this line, something like `Console.WriteLine(distance + " furlongs");`  (except find out what units are actually being used)